framework/contrib/hit/hit.h

```diff
@@ -0,0 +1,9 @@
+
+#ifndef HIT_HIT
+#define HIT_HIT
+
+#include "parse.h"
+
+#include <fstream> // TODO: remove this - hit parser doesn't need it at all
```

Comment:
Why add it then?

Reply:
Lots of moose and app files were borrowing the fstream include from the old getpot parser's header :-(.  So removing this will be a fix for another day.